pkgpanda/test_check.py

```diff
@@ -21,13 +30,16 @@
 
 
 def test_check_target_list():
+    root_dir = resources_test_dir("opt/mesosphere")
+    repo_dir = resources_test_dir("opt/mesosphere/packages")
     output = check_output([
         'pkgpanda',
         'check',
         '--list',
-        '--root', resources_test_dir("opt/mesosphere"),
-        '--repository', resources_test_dir("opt/mesosphere/packages")], stderr=STDOUT)
-    assert output.decode() == list_output
+        '--root', root_dir,
+        '--repository', repo_dir], stderr=STDOUT)
+    output = output.decode().replace('\r', '')   # Eliminate the CRs that windows inserts
```

Comment:
See above comment about splitting out tests. We shouldn't be running code intended to accommodate Windows if we're on Unix. This only works because the check output coincidentally doesn't include carriage returns.